src/wizard/android/index.md

```diff
@@ -32,49 +26,38 @@ plugins {
-
-```groovy
-// Make sure mavenCentral is there.
-repositories {
-    mavenCentral()
-}
-
-// Enable Java 1.8 source compatibility if you haven't yet.
-android {
-    compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
-    }
-}
-
-// Add Sentry's SDK as a dependency.
-dependencies {
-    implementation 'io.sentry:sentry-android:{{ packages.version('sentry.java.android', '4.0.0') }}'
-}
-```
+The plugin version `{{ packages.version('sentry.java.android.gradle-plugin', '3.0.0') }}` will automatically add the Sentry Android SDK (version `{{ packages.version('sentry.java.android', '4.2.0') }}`) to your app.
 
 ## Connecting the SDK to Sentry
 
-The code snippet below includes the DSN, which tells the SDK to send the events to this project.
-
-Add your DSN to the manifest file.
+Configuration is done via the application `AndroidManifest.xml`. The code snippet below includes the DSN, which tells the SDK to send the events to this project.
 
+Here's an example config which should get you started:
 ```xml {filename:AndroidManifest.xml}
 <application>
-    <meta-data android:name="io.sentry.dsn" android:value="___PUBLIC_DSN___" />
-    <!-- Set tracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
-       We recommend adjusting this value in production. -->
-    <meta-data android:name="io.sentry.traces.sample-rate" android:value="1.0" />
-    <!-- Enable user interaction tracing to capture transactions for various UI events (such as clicks or scrolls). -->
-    <meta-data android:name="io.sentry.traces.user-interaction.enable" android:value="true" />
+  <!-- Required: set your sentry.io project identifier (DSN) -->
+  <meta-data android:name="io.sentry.dsn" android:value="___PUBLIC_DSN___" />
+
+  <!-- enable automatic breadcrumbs for user interactions (clicks, swipes, scrolls) -->
+  <meta-data android:name="io.sentry.traces.user-interaction.enable" android:value="true" />
+  <!-- enable screenshot for crashes -->
+  <meta-data android:name="io.sentry.attach-screenshot" android:value="true" />
+  <!-- enable view hierarchy for crashes -->
+  <meta-data android:name="io.sentry.attach-view-hierarchy" android:value="true" />
+
+  <!-- enable the performance API by setting a sample-rate, adjust in production env -->
+  <meta-data android:name="io.sentry.traces.sample-rate" android:value="1.0" />
+  <!-- enable profiling when starting transactions, adjust in production env -->
+  <meta-data android:name="io.sentry.traces.profiling.sample-rate" android:value="1.0" />
 </application>
 ```
 
+Under the hood Sentry uses a `ContentProvider` to initalize the SDK based on the values provided above. This way the SDK can capture important crashes and metrics right from the app start.
+
+Additional options can be found [on our dedicated options page](/platforms/android/configuration/options/).
```

Comment:
I think you have to use the full url here and in the paragraph below, because relative links do not work with the wizard.